bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+num_users_updated = 0
+batch_size = 10000
```

Comment:
How did you decide on this batch size?

Reply:
Good question.  JoshL shared that the team typically uses a batch_size of 10k.

Reply:
However I have seen a batch/transaction size of 2500 and 5000 used.

Reply:
Yeah I noticed that too but I don't have a strong opinion either way.